config/default/manager/manager.yaml

```diff
@@ -21,14 +21,15 @@ data:
   RANGE_END: 02:FF:FF:FF:FF:FF
 ---
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
   name: mac-controller-manager
   namespace: system
   labels:
     control-plane: mac-controller-manager
     controller-tools.k8s.io: "1.0"
 spec:
+  serviceName: "kubemacpool-service"
```

Comment:
why did you use ""?

Reply:
this is string type
https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L152